### ISSUE TITLE
waveform doesnt show anything

### ISSUE THREAD

**Issue by MSOB7YY** [SimformSolutionsPvtLtd/audio_waveforms#142]
i want to display waveform for files on the device already, i have copied the example` chat_bubble.dart` file and provided the file path from the main widget, nothing shows in the widget.
only empty data

here is the `chat_bubble.dart` only modification
note: code doesnt proceed till the last print statement unless commenting the selected lines (i already have a sample file in assets folder)
![Screenshot_20221221_103922](https://user-images.githubusercontent.com/85245079/208998940-677bea8e-1d01-456d-a065-fd927aed60fd.png)

the path and uri is printed succesfully in the console, but for some reason it doesnt extract anything, is it something related to path name?
![Screenshot_20221221_103744](https://user-images.githubusercontent.com/85245079/208998893-b82fde75-ca4f-4867-b7d3-e257d966f81f.png)

- Device: [Pixel 5 API 31 (x64 emulator) ]

**Comment by ujas-m-simformsolutions** [comment]
@MSOB7YY Yes it is related to path name. Your path contains spaces in it so I would suggest remove spaces from actual file and then again to extract waveform data. Also you can see your `maxDuration` is -1 so it also couldn't initialize the player.

**Comment by MSOB7YY** [comment]
i tried with another file with no spaces, it worked. does this mean that reading files with spaces is impossible?

**Comment by ujas-m-simformsolutions** [comment]
@MSOB7YY No that's not it because we are using ExoPlayer for audio player and MediaExtractor for decoding process on android, sometimes both fails to initialize and sometimes only MediaExtractor fails.

We will see how should we handle path with spaces.

**Comment by MSOB7YY** [comment]
thank you so much for your hard effort 🙏
